doc/contribute_models.md

```diff
@@ -37,9 +37,7 @@ Note that we set the environment variable `PYTHONPATH` so that we can directly t
 
 Open the browser and go to http://localhost:8888, it's a Jupyter notebook environment, you can see your model development directory `my_awesome_model` together with SQLFlow's basic tutorials.
 
-<p align="center">
-<img src="figures/jupyter_develop.jpg">
-</p>
+![](figures/jupyter_develop.jpg)
```

Comment:
It looks we need to display the image in the middle(center) of line.

Reply:
Yes, I tried some way but failed, it seems we can't add css.
And if we add <p align="center"> img just disappeared